applications/nrf5340_audio/src/bluetooth/bt_management/bt_mgmt.h

```diff
@@ -52,6 +52,8 @@ enum bt_mgmt_scan_type {
 	BT_MGMT_SCAN_TYPE_BROADCAST = 2,
 };
 
+#define BRDCAST_ID_NOT_USED 0x1000000
```

Comment:
Is there a define for this somewhere in host?

Reply:
No, I created that, but there is something similar in the shell\bap.c #define INVALID_BROADCAST_ID (BT_AUDIO_BROADCAST_ID_MAX + 1), I can use that instead. We still have to define it locally though, and it is the same value, but it will look less magic